cpplint.py

```diff
@@ -6226,12 +6226,14 @@ def CheckForIncludeWhatYouUse(filename, clean_lines, include_state, error,
                        for item in sublist])
 
   # All the lines have been processed, report the errors found.
-  for required_header_unstripped in sorted(required, key=required.__getitem__):
-    template = required[required_header_unstripped][1]
-    if required_header_unstripped.strip('<>"') not in include_dict:
-      error(filename, required[required_header_unstripped][0],
+  for header in sorted(required, key=required.__getitem__):
+    template = required[header][1]
+    if (header.strip('<>"') not in include_dict
+            and not (header.strip('<>"')[0] == 'c'
+                     and (header.strip('<>"')[1:] + '.h') in include_dict)):
```

Comment:
Call str.strip() once instead of three times.
```suggestion
    header_stripped = header.strip('<>"')
    if (header_stripped not in include_dict
            and not (header_stripped[0] == 'c'
                     and (header_stripped[1:] + '.h') in include_dict)):
```